code/game/gamemodes/steal_items.dm

```diff
@@ -6,10 +6,11 @@
 #define THEFT_FLAG_UNIQUE 2
 
 /datum/theft_objective
-	var/name=""
+	var/name = ""
 	var/typepath=/atom
-	var/list/protected_jobs=list()
-	var/flags=0
+	var/list/protected_jobs = list()
+	var/list/altitems = list()
+	var/flags = 0
 
 /datum/theft_objective/proc/check_completion(var/datum/mind/owner)
```

Comment:
Wait, you can steal money? Why is that not an objective? That would be great to steal credits from other peoples accounts